packages/openneuro-components/src/search-page/SearchPage.tsx

```diff
@@ -39,10 +43,43 @@ export const SearchPage = ({ portalContent }: SearchPageProps) => {
         ) : null}
 
         <section className="search">
-          <div className="container">
-            <div className="grid grid-start">
-              <div className="col col-4">facets Todo</div>
-              <div className="col col-8">results todo</div>
+          <div className="grid grid-start">
+            <div className="col col-12">
+              <h1>
+                {portalContent ? 'Search MRI Portal' : 'Search all Dataset'}
+              </h1>
+            </div>
+
+            <div className="col col-12">
+              <div className="grid grid-between grid-nogutter">
+                <div
+                  className="col"
+                  style={{
+                    maxWidth: '500px',
+                  }}></div>
+                <div className="col  col-center results-count">
+                  <b>
+                    100 Datasets found for "<span>MRI</span>"
+                  </b>
+                </div>
+                <div className="col col-center">
+                  <div className="search-sort">
+                    <SortBy {...SortBy.args} />
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div className="col search-facet-wrapper">
+              <FacetExample {...FacetExample.args} />
```

Comment:
This and SearchResults would be better as a render props, meaning that we should pass <FacetExample /> and all other search control features on the left panel in via a function prop.